src/main/java/com/google/edith/interfaces/StoreReceiptInterface.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.edith.interfaces;
+
+import com.google.edith.servlets.Receipt;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+
+/** Operations for handling parsing receipts. */
+public interface StoreReceiptInterface {
```

Comment:
Interfaces shouldn't have "Interface" in their names. Interfaces are the primary surface for this service, so they should have the primary name: "StoreReceiptService" or something like that. The implementation can bear additional overhead in the name because it is private and not particularly exposed to the consumer. (i.e. StoreReceiptServiceImpl)

Reply:
I am adding an issue to change all the Interface names. I will create a new PR to address these all.